### PR TITLE
chore: Prepare 1.3.0-beta3 release

### PR DESCRIPTION
**Pull request by pawels-optimizely**

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 Changes that have landed but are not yet released.	Changes that have landed but are not yet released.
 
+## [1.3.0-beta3] - June 30th, 2020
+## New Features
```
Comment (aliabbasrizvi) [inline CHANGELOG.MD:11]: Is this a bug or new feature?

File: CHANGELOG.MD [hunk 2]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Provide syncronization for Notification Center Cache ([#273](https://github.com/optimizely/go-sdk/pull/273))
+
+
 ## [1.3.0-beta2] - June 17th, 2020
```
Comment (aliabbasrizvi) [inline CHANGELOG.MD:15]: Similarly, is this bug or new feature?

File: CHANGELOG.MD (hunk 1 above)
Comment (pawels-optimizely) [reply CHANGELOG.MD]: fixed

File: CHANGELOG.MD (hunk 2 above)
Comment (pawels-optimizely) [reply CHANGELOG.MD:15]: fixed

**Pull request merged**